### PR TITLE
Liveness and readiness check

### PR DESCRIPTION
**Pull request by jasonblais**
Summary of ways to perform a liveness and readiness check of a Mattermost server.

Notes:
 - We can later consider adding more documentation for the Mattermost Probe project, but have linked to it for now: https://github.com/csduarte/mattermost-probe
 - The ``/ping`` APIv4 Endpoint section should be reviewed for accuracy by a dev.

### CODE REVIEW

**Comment by jasonblais** [comment]
It's a single word, thanks Lindy!

File: source/administration/liveness-check.rst [hunk 1]
```diff
@@ -0,0 +1,39 @@
+Liveness and Readiness Check
+=============================
+
+This page describes how to configure liveness and readiness probes for a Mattermost server.
+
+Before you begin, you should have a running Mattermost server. If you don't, you can `install Mattermost on various distributions <https://docs.mattermost.com/guides/administrator.html#installing-mattermost>`_ or deploy a `Kubernetes cluster with Minikube <https://github.com/mattermost/mattermost-kubernetes>`_.  Note that `highly available Mattermost cluster <https://docs.mattermost.com/deployment/cluster.html>`_ is available on `Enterprise Edition E20 <https://about.mattermost.com/pricing/>`_.
```
Comment (jwilander) [inline source/administration/liveness-check.rst]: I think it should be `is available in [EE]` not `is available on [EE]`

File: source/administration/liveness-check.rst [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  status, err := Client.GetPing()
+
+Mattermost Probe
+-----------------
+
+The `Mattermost Probe <https://github.com/csduarte/mattermost-probe>`_ constantly pings a Mattermost server using a variety of probes.
+
+These probes can be configured to verify core features, including sending and receiving messages, joining channels, pinging a login page, and searching of users and channels.
+
+The project is contributed by the Mattermost open-source community. Suggestions and contributions for the project are welcome.
```
Comment (lfbrock) [inline source/administration/liveness-check.rst]: typically "open source" without hyphen I think

**Comment by jasonblais** [comment]
Thanks @jwilander and @lfbrock! Fixed the grammar mistakes + added to Administration TOC, below `Backup and Disaster Recovery`.

**Pull request merged**